.github/CODEOWNERS

```diff
@@ -9,6 +9,8 @@ python/dask_cudf/  @rapidsai/cudf-dask-codeowners
 
 #cmake code owners
 cpp/CMakeLists.txt               @rapidsai/cudf-cmake-codeowners
+cpp/tests/CMakeLists.txt         @rapidsai/cudf-cmake-codeowners
```

Comment:
Are you planning on reverting this change? I presumed you added it to get a `cudf-cmake-codeowners` ping. We removed this from code owners to reduce amount of reviews for tests/benchmarks since they generally are just adding new files

Reply:
I didn't realize that was an intentional removal. I'll drop them.

Reply:
Removed.